bellcom/securipax
# Add pre/post script calling.

How to check steps:
* go to page `/admin/config/securipax_site2html/export`
* specify command to scripts
* save settings
* run export
You should see output execution of  pre/post commands
